steam_workshop/config.json

```diff
@@ -1,9 +1,9 @@
 {
   "steamPublishedID": "2843056085",
-  "title": "Downfall Relic Stats compatibility",
-  "description": "Adds Relic Stats compatibility to the Downfall Expansion.",
+  "title": "Downfall Relic Stats",
+  "description": "",
```

Comment:
please undo this change

Reply:
ive put the title back, but its basically never worth putting anything in the description since itll just override whatever u do in steam